packages/remix-dev/compiler/compiler.ts

```diff
@@ -108,12 +113,12 @@ export let create = async (ctx: Context): Promise<Compiler> => {
     options.onManifest?.(manifest);
     writes.manifest = writeManifest(ctx.config, manifest);
 
-    // server compilation
```

Comment:
Could you restore this comment?

Reply:
Okay, this is really my oversight.

Reply:
In fact, a better solution is that I think metafile should be turned off by default under dev.